scenarios/evaluate/README.md

```diff
@@ -0,0 +1,50 @@
+| simulate_adversarial.ipynb            | X           | X         |                     |      |          | X                      |             |                     |                |                      |
+| simulate_conversation_starter.ipynb   |             | X         | X                   |       |         | X                      |             |                     |                |                      |
+| simulate_input_index.ipynb            |             | X         |                     | X     |          | X                      |             |                     |                |                      |
+| simulate_input_text.ipynb             |             | X         |                     |       | X        | X                     |             |                     |                |                      |
+| evaluate_endpoints.ipynb              |             |           |                     |      |          | X                      |            | X                    |                |                      |
+| evaluate_app.ipynb                    |             |           |                     |       |         |                       | X           | X                    |                |                      |
+| evaluate_qualitative.ipynb            |             |           |                     |       |         | X                      |            | X                    |                |                      |
+| evaluate_custom.ipynb                 |             |           |                     |       |         | X                      |            |                     | X               |                      |
+| evaluate_quantitative.ipynb            |             |           |                     |       |         | X                      |             |                     |               | X                     |
+| evaluate_safety_risk.ipynb            | X           |           |                     |       |          | X                     |             |                     |                |                      |
+| simulate_and_evaluate_endpoint.py      |             | X         |                     |      | X        | X                     |             | X                    |                |                    |
+
+
+
+### Pre-requisites
+To use the `azure-ai-evaluation` SDK, install with```pythonpip install azure-ai-evaluation```Python 3.8 or later is required to use this package.- See our Python reference documentation for our `azure-ai-evaluation` SDK[here](https://aka.ms/azureaieval-python-ref) for more granular details oninput/output requirements and usage instructions.- Check out our Github repo for `azure-ai-evaluation` SDK [here](    https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/evaluation/azure-ai-evaluation). 
```

Comment:
typo: *on input/output

Note: this is a note-to-self to add cspell to our precommit to catch these things - this can go in our samples backlog